src/sentry/tasks/statistical_detectors.py

```diff
@@ -348,14 +347,14 @@ def detect_transaction_trends(
         project_option=InternalProjectOptions.TRANSACTION_DURATION_REGRESSION,
     )
 
-    trends = EndpointRegressionDetector.detect_trends(projects, start)
+    trends = EndpointRegressionDetector.detect_trends(projects, start_time)
     trends = EndpointRegressionDetector.get_regression_groups(trends)
-    trends = EndpointRegressionDetector.redirect_resolutions(trends, start)
-    trends = EndpointRegressionDetector.redirect_escalations(trends, start)
+    trends = EndpointRegressionDetector.redirect_resolutions(trends, start_time)
+    trends = EndpointRegressionDetector.redirect_escalations(trends, start_time)
     trends = EndpointRegressionDetector.limit_regressions_by_project(trends)
 
     delay = 12  # hours
-    delayed_start = start + timedelta(hours=delay)
+    delayed_start = start_time + timedelta(hours=delay)
```

Comment:
```suggestion
    delayed_start = start + timedelta(hours=delay)
```

I think you want to add the `datetime` and timedelta.